pkg/policy/api/ingress.go

```diff
@@ -30,12 +30,7 @@ import (
 //   the effects of any Requires field in any rule will apply to all other
 //   rules as well.
 //
-// - For now, combining ToPorts, FromCIDR, and FromEndpoints in the same rule
-//   is not supported and any such rules will be rejected. In the future, this
-//   will be supported and if multiple members of this structure are specified,
-//   then all members must match in order for the rule to take effect. The
-//   exception to this rule is the Requires field, the effects of any Requires
-//   field in any rule will apply to all other rules as well.
+// Combining FromCIDR and FromEndpoints in the same rule is not allowed.
```

Comment:
Sorry, late mention but while we're fixing up this comment I think that `FromEndpoints`, `FromCIDR`, `FromCIDRSet`, `FromEntities` are mutually exclusive. I propose wording like this in an additional paragraph:

```
// - FromEndpoints, FromCIDR, FromCIDRSet and FromEntities are mutually
//   exclusive. Only one of these members may be present within an individual rule.
```

Reply:
^ Can be a follow-up.

Reply:
Done